app/parse.py

```
@@ -14,10 +22,108 @@ class Product:
     rating: int
     num_of_reviews: int
 
+def scrape_products_from_page(driver: webdriver.Chrome, url: str, product_selector: str, use_more_button: bool = False) -> List[Product]:
+    driver.get(url)
+    wait = WebDriverWait(driver, 10)
+
+    try:
+        cookies = driver.find_element(By.CSS_SELECTOR, ".accept-cookies-button")
+        if cookies:
+            cookies[0].click()
+    except Exception as e:
```

Broad exception used for cookie handling hides real issues and can cause long unexpected waits. Change the broad `except Exception as e:` (line 33) to catching the specific exceptions you expect (e.g., `NoSuchElementException` or `TimeoutException`), and log a clear message. This prevents silent failures and matches the requirement to handle cookie consent safely.